libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -15,7 +15,7 @@ interface TicketReplyFormValues {
 
 /* This validates the form for the first textarea in the form.
 This will also show the red required text underneath the textarea.
-If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component. 
+If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component.
```

Comment:
**Superfluous Changes.** Restore space removed upon save of file in my code editor.

```suggestion
If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component. 
```

Reply:
Same thing, again.

```suggestion
If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component. 
```

Reply:
GitHub won't let me do this. But, my editor **should** be removing stray spaces. So, I'm not fighting this extra change in the diff.